web/src/pages/Req_Detail.tsx

```diff
@@ -4,49 +4,62 @@ import Document from "../components/Doc_Review";
 import Req_Button from "../components/Req_Button";
 import { IoIosArrowBack } from "react-icons/io";
 import { useNavigate } from "react-router-dom";
+import { useLazyLoadQuery } from "react-relay";
+import graphql from "babel-plugin-relay/macro";
+
+import { ReqDetailQuery, ReqDetailQuery$data } from "./__generated__/ReqDetailQuery.graphql";
 
 const ReqDetail = (): JSX.Element => {
   // Navigation - Go back to Req_Revision
   const navigate = useNavigate();
 
-  // Request - Info
-  const exampleRequest = {
-    titulo: "Arte Típico",
-    autor: "Susana Horia",
-    img: "https://infolibros.org/wp-content/uploads/2021/06/Libros-de-Artes-Visuales.jpg?ezimgfmt=ng%3Awebp%2Fngcb33%2Frs%3Adevice%2Frscb33-1",
-  };
+  // Request - Info / Detail
+  const data: ReqDetailQuery$data = useLazyLoadQuery<ReqDetailQuery>(
+    graphql`
+      query ReqDetailQuery($application_id: ID!) {
+        application(id: $application_id) {
+          title
+          image
+          description
+          support
+          user {
+            name
+          }
+        }
+        applicationdocuments(application_id: $application_id) {
+          file_name
+          url
+          updated_at
+        }
+      }
+    `,
+    { application_id: "8bc874cc-e021-4504-ac90-a473e57e999c" },
+  );
```

Comment:
Add document list object within application model and handle fetching inside the query.